test/acceptance/tests/basic/basic_test.go

```diff
@@ -19,27 +19,12 @@ import (
 	"github.com/gruntwork-io/terratest/modules/random"
 	"github.com/gruntwork-io/terratest/modules/shell"
 	"github.com/gruntwork-io/terratest/modules/terraform"
-	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/hashicorp/serf/testutil/retry"
 	"github.com/hashicorp/terraform-aws-consul-ecs/test/acceptance/framework/helpers"
 	"github.com/hashicorp/terraform-aws-consul-ecs/test/acceptance/framework/logger"
 	"github.com/stretchr/testify/require"
 )
 
-// Test the validation that if TLS is enabled, Consul's CA certificate must also be provided.
-func TestValidation_CACertRequiredIfTLSIsEnabled(t *testing.T) {
-	t.Parallel()
-	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
-		TerraformDir: "./terraform/ca-cert-validate",
-		NoColor:      true,
-	})
-	t.Cleanup(func() {
-		_, _ = terraform.DestroyE(t, terraformOptions)
-	})
-	_, err := terraform.InitAndPlanE(t, terraformOptions)
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "ERROR: consul_server_ca_cert_arn must be set if tls is true")
-}
```

Comment:
non-blocking / tech debt: This is long overdue, but it would be good to move the TestValidation cases to a separate file or separate test package from the TestBasic acceptance test.

Reply:
Fixed it